cmake/projects/aws-sdk-cpp/hunter.cmake

```diff
@@ -9,6 +9,17 @@ include(hunter_cmake_args)
 include(hunter_download)
 include(hunter_pick_scheme)
 
+hunter_add_version(
```

Comment:
could you reorder the versions for the newest to be at the bottom. Some tools don't produce nice git diffs when the newest change is on top